src/Mwi.LoanPay/Apis/LoanPayApi.cs

```diff
@@ -407,13 +436,29 @@ public LoanPayApi(HttpClient httpClient, IEnvironmentManager environmentManager)
                 };
             }
 
+            // Cancelling a pending payment returns an empty body if it succeeds, so we shouldn't try and parse anything and just exit.
+            if (response.IsSuccessStatusCode)
```

Comment:
What statuses are you expecting to get back? "304 - NotModified" is not included in the range of "Success" status codes.

Reply:
https://mpay-api.mwamplifi.com/swagger/ui/index#!/LoanPay/LoanPay_CancelPayment

![image](https://user-images.githubusercontent.com/57642738/141192934-015b456c-baff-45c2-a022-93ce17a38df1.png)


200 OK on success